### PR TITLE
 feat: getAddresses method should expect the account xpub derivation and receive the changeDerivation

### PR DESCRIPTION
**Pull request by pedroferreira1**

### CODE REVIEW

File: src/helpers.js [hunk 1]
```diff
@@ -405,9 +406,12 @@ const helpers = {
       network.setNetwork(networkName);
     }
 
+    // The xpub is in the account derivation step, we must derive until the change path
+    const derivedXpub = xpub.derive(changeDerivation);
```
Comment (msbrogli) [inline src/helpers.js:410]: I don't get why we need to derive here.

File: src/utils/wallet.ts [hunk 2]
```diff
@@ -211,7 +212,7 @@ const wallet = {
    * @memberof Wallet
    * @inner
    */
-  getAddresses(xpubkey: string, startIndex: number, quantity: number, networkName: string = 'mainnet'): Object {
+  getAddresses(xpubkey: string, changeDerivation: number, startIndex: number, quantity: number, networkName: string = 'mainnet'): Object {
```
Comment (msbrogli) [inline src/utils/wallet.ts:215]: I feel I've already seen a similar change in this PR. Why do we need the `changeDerivation` here?

**Review by msbrogli — CHANGES_REQUESTED** [review]
I don't understand what these changes are doing and why they are useful.

**Comment by pedroferreira1** [comment]
This PR is not needed anymore, as discussed privately.

**Pull request closed without merging**